content/en/guides/features/rendering-modes.md

```diff
@@ -24,18 +24,24 @@ export default {
 ```
 
 <base-alert type="info">
+
 You do not need to add this to your nuxt config in order for universal mode to be applied as the default mode is universal.
+
 </base-alert>
 
 ## SPA
 
-`mode: 'spa'`: No server-side rendering, only client-side rendering.
+<base-alert type="warning">
+
+deprecated `mode: 'spa'` please use `ssr: false`.
+
+</base-alert>
 
-With Single Page Applications there is no server-side rendering, only client-side rendering. Client side rendering means rendering the content in the browser using JavaScript. Instead of getting all of the content from the HTML we just get a basic HTML document with a JavaScript file that will then render the rest of the site using the browser.
+With Single Page Applications there is no server-side rendering, only client-side rendering. Client side rendering means rendering the content in the browser using JavaScript. Instead of getting all of the content from the HTML we just get a basic HTML document with a JavaScript file that will then render the rest of the site using the browser. To deploy a single page application set ssr to `false` and then use the `build` command to build your application.
```

Comment:
For deploy, can we instead link to deployment-targets/spa?

Reply:
ok done